app.js

```diff
@@ -9,10 +13,14 @@ const {
 const express = require("express");
 const app = express();
 
+app.use(express.json());
+
 app.get("/api/topics", getTopics);
 
 app.get("/api/articles/:article_id", getArticle);
 
+app.get("/api/articles", getAllArticles);
+
 app.get("*", invalidPathHandler);
 
 app.use(handleCustomError);
```

Comment:
Please add an error handling middleware to the end to handle internal server errors.